src/content/guidelines/color/overview.md

```diff
@@ -0,0 +1,185 @@
+---
+label: Color is a foundational element of Carbon. Careful application of color drives consistency, engagement, and focus for all digital interfaces.  
+title: Color
+tabs: ['Overview', 'Usage']
+---
+
+<anchor-links>
+<ul>
+    <li><a href="#resources">Resources</a></li>
+    <li><a href="#introduction">Introduction</a></li>
+    <li><a href="#ibm-color-anatomy">IBM Color anatomy</a></li>
+    <li><a href="#tokens">Tokens</a></li>
+    <li><a href="#themes">Themes</a></li>
+    <li><a href="#interaction-states">Interaction states</a></li>
+    <li><a href="#digital-accessibility">Digital accessibility</a></li>
+</ul>
+</anchor-links>
+
+## Resources
+
+<grid-wrapper col_lg="8" flex="true" bleed="true">
+<clickable-tile
+    title="Color palettes for Adobe™(.ase)"
+    href="https://w3.ibm.com/design/language//IBM-color-palettes.zip"
+    type="resource"
+    >
+    <img src="images/ase.png" alt="Color palettes for Adobe" />
+</clickable-tile>
+</grid-wrapper>
+
+## Introduction
```

Comment:
Added back in, though the reason I changed it in the first place was because this language is very IBM-focused, while this page should be applicable to all Carbon users, including those outside IBM. This page isn't really about the color palette itself, it's about _how color works in Carbon_. . .